internal/store/logs.go

```diff
@@ -20,13 +20,28 @@ import (
 	"io"
 
 	atlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	om "github.com/mongodb/go-client-mongodb-ops-manager/opsmngr"
 	"github.com/mongodb/mongocli/internal/config"
 )
 
 type LogsDownloader interface {
 	DownloadLog(string, string, string, io.Writer, *atlas.DateRangetOptions) error
 }
 
+type Logs interface {
```

Comment:
[nit] one line interfaces should be named as what they do, like
```suggestion
type LogCollector interface {
```